src/applications/simple-forms/40-0247/tests/unit/helpers.unit.spec.js

```diff
@@ -24,36 +24,37 @@ describe('trackNoAuthStartLinkClick', () => {
   });
 });
 
-describe('getInitialData', () => {
-  it('returns mockData if environment is localhost and Cypress is not running', () => {
-    const mockData = { foo: 'bar' };
-    const environment = {
-      isLocalhost: () => true,
-    };
-    const result = getInitialData({ mockData, environment });
-    expect(result).to.deep.equal(mockData);
-  });
-
-  it('returns undefined if environment is not localhost', () => {
-    const mockData = { foo: 'bar' };
-    const environment = {
-      isLocalhost: () => false,
-    };
-    const result = getInitialData({ mockData, environment });
-    expect(result).to.be.undefined;
-  });
-
-  it('returns undefined if Cypress is running', () => {
-    const mockData = { foo: 'bar' };
-    const environment = {
-      isLocalhost: () => true,
-    };
-    window.Cypress = true;
-    const result = getInitialData({ mockData, environment });
-    expect(result).to.be.undefined;
-    window.Cypress = undefined;
-  });
-});
+// ADD THIS BACK IN WHEN REMOVING THE PROD FLAG FOR MBMS-64950
```

Comment:
could we just leave this as is without commenting it out?

Reply:
I'm not sure because this test fails due to the naming convention around the word 'environment'. We were planning to uncomment this during the flag removal story that follows this bug.

Reply:
ah ok. I see the problem. you could optionally do
`import env from 'platform/utilities/environment';`
and then use `env.isProduction()`, so it doesn't conflict with the passed in `environment` variable. 
it just seems like it would be nice if we didn't have to go through loops of commenting out and reenabling tangent code, and could just focus on the new code.

Reply:
or maybe better, to change the initalData one to:
```js
export function getInitialData({ mockData, environment: env }) {
  return !!mockData && env.isLocalhost() && !window.Cypress
    ? mockData
    : undefined;
}
```

Reply:
Gotcha! I'll give that a shot. Thanks!

Reply:
This worked great - thanks!